myapp/spec/controllers/tasks_controller_spec.rb

```diff
@@ -48,4 +51,19 @@
     expect(response.status).to eq 302
     expect(Task.count).to eq 0
   end
+
+  it 'Sort by created_at desc' do
+    @task1 = FactoryBot.create(:task, name: 'task1', created_at: Time.zone.tomorrow)
+    @task2 = FactoryBot.create(:task, name: 'task2', created_at: Time.zone.yesterday)
+
+    get :index
+    expect(response.status).to eq 200
+
+    res_str = response.body
+
+    task1_index = res_str.index('task1')
+    task2_index = res_str.index('task2')
+
+    expect(task1_index).to be < task2_index
```

Comment:
wow. I didn't know you can do comparison like that. 

Well it work. 
But I'm sorry to tell you that the english instruction got wrong translation on it.

It was suppose to instruct you to write "feature spec" for this step.
https://relishapp.com/rspec/rspec-rails/docs/feature-specs/feature-spec

English | Japanese
----- | -----
![image](https://user-images.githubusercontent.com/10822260/122311504-4f039b80-cf4d-11eb-9d68-cf9d470b91ec.png) | ![image](https://user-images.githubusercontent.com/10822260/122311547-693d7980-cf4d-11eb-8525-e62bd3c77cde.png)

With capybara's API, it's easier to do the comparison. 
you can parse the text on the title field and compare it with the expected order of the task.

ref(if needed): 

https://github.com/Fablic/training/pull/405/files#diff-82689603be06b3081543d76c323b7476b917358f81297e164b83cdf1ebbe8359R66-R83

Reply:
Done